tools/bin/ecflow/processes/main/post_processing/sm_analysis.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+"""
+creating sm percentiles
+usage: <python> <sm_plot.py> <configuration.cfg>
+
+Reads in a netcdf file converted from VIC fluxes.
+Extracts one day's data based on date in config file.
+Reads in saved cdfs.
+Interpolates every value's percentile
+based on where it falls relative to historic range.
+Save as new netcdf file.
+"""
+from datetime import datetime, timedelta
+import numpy as np
+import pandas as pd
+import xarray as xr
+import os
+import time
+import math
+import gc
+import argparse
+from scipy.interpolate import interp1d
+from netCDF4 import Dataset
+from tonic.io import read_config
+from collections import OrderedDict
+
+# read in configuration file
+parser = argparse.ArgumentParser(description='Calculate SWE percentiles')
+parser.add_argument('config_file', metavar='config_file',
+                    help='configuration file')
```

Comment:
More descriptive help than just a longer version of the variable name? Maybe point to a sample config file?